pkg/policy/l4.go

```diff
@@ -1144,8 +1143,8 @@ type L4PolicyMap interface {
 	IngressCoversContext(ctx *SearchContext) api.Decision
 	EgressCoversContext(ctx *SearchContext) api.Decision
 	ForEach(func(l4 *L4Filter) bool)
-	Equals(t *testing.T, bMap L4PolicyMap) bool
-	Diff(t *testing.T, expectedMap L4PolicyMap) string
+	Equals(bMap L4PolicyMap) bool
```

Comment:
The reason the 'testing.T` parameter is added to these methods is to ensure they are not used in production code. There, at least, be a convention that preserves this intent.
```suggestion
	TestingOnlyEquals(bMap L4PolicyMap) bool
```